MQuince/MQuince.Application/App.cs

```diff
@@ -44,13 +44,11 @@ private string ExtractArgument(string argument)
         }
 
         public IUserService GetUserService()
-            => new UserService(this.GetUserRepository());
+            => new UserService(this.GetPatientRepository(),this.GetAdminRepository());
```

Comment:
Pull the current version of develop branch, user repositories are now merged in one user repository.